doctr/__main__.py

```diff
@@ -134,6 +135,10 @@ def get_parser(config=None):
     if we do not appear to be on Travis.""")
     deploy_parser_add_argument('deploy_directory', type=str, nargs='?',
         help="""Directory to deploy the html documentation to on gh-pages.""")
+    deploy_parser_add_argument('--dkenv', type=str, metavar="ENVVAR",
```

Comment:
We should be able to detect this automatically.

Reply:
Maybe, personally I think the interface is much simpler and more flexible if the user picks the environment variable name(s) themselves.

Reply:
We can make a flag if the user wants to change it, but there should be a default. It should be possible to just use the variable names we already use, and detect based on the content if it is pointing to a file or if it is a private key. Most users don't really care how doctr works. They just do whatever it says to do and if it pushes their stuff up to gh-pages they are happy.